### ISSUE TITLE
Would you share the model file for the imagenet's top 500 classes?

### ISSUE THREAD

**Issue by JoyHuYY1412** [arthurdouillard/incremental_learning.pytorch#23]

**Comment by JoyHuYY1412** [comment]
I am not so sure, it seems it needs to train around 10 days to get the first model?

**Comment by arthurdouillard** [comment]
I'm very sorry but because of a stupid manipulation when I release my code open-source, I've deleted a lot of my pre-trained weights, including those of ImageNet1000.

It should take between 5 to 10 days indeed depending on your GPU.

**Comment by JoyHuYY1412** [comment]
> I'm very sorry but because of a stupid manipulation when I release my code open-source, I've deleted a lot of my pre-trained weights, including those of ImageNet1000.
> 
> It should take between 5 to 10 days indeed depending on your GPU.

Thank you for your reply!

**Comment by JoyHuYY1412** [comment]
I have one more small question, could you please give me some advice? 
I run `python3 -minclearn --options options/bic/bic_cifar100.yaml options/data/cifar100_3orders.yaml --initial-increment 50 --increment 10 --fixed-memory --device 6 --label bic_cifar100_50_step_10_memory_20 --data-path ./data -save task --no-benchmark -memory 2000` like this trying to use your BiC model. But the average accuracy is a little bit low (51.02). Do I need to do some adjustment?

**Comment by arthurdouillard** [comment]
It's indeed much lower than my paper's results of BiC (~56).

I'll look into it this week, but your command seems fine.

**Comment by JoyHuYY1412** [comment]
> It's indeed much lower than my paper's results of BiC (~56).
> 
> I'll look into it this week, but your command seems fine.

Thanks again!
